python/cudf/cudf/core/column/numerical_base.py

```diff
@@ -246,12 +246,21 @@ def corr(self, other: NumericalBaseColumn) -> float:
         return cov / lhs_std / rhs_std
 
     def round(
-        self, decimals: int = 0, how: str = "half_even"
+        self,
+        decimals: int = 0,
+        how: Literal["half_even", "half_up"] = "half_even",
     ) -> NumericalBaseColumn:
         if not cudf.api.types.is_integer(decimals):
-            raise TypeError("Values in decimals must be integers")
-        """Round the values in the Column to the given number of decimals."""
-        return libcudf.round.round(self, decimal_places=decimals, how=how)
+            raise TypeError("decimals must an integer")
```

Comment:
Just suggesting to be consistent with a previous review.
```suggestion
            raise TypeError(f"Argument 'decimals' must an integer")
```